Core/MajorTrackerNetworks.swift

```diff
@@ -30,14 +30,3 @@ struct MajorTrackerNetworks {
         "oracle.com"
     ]
 }
-
-extension Tracker {
```

Comment:
Can we keep this? This major network logic has expanded to include percentages etc. Happy to make it an extension on the inner type SiteRating.Tracker for now & I'll refactor on my branch

Reply:
This is now available as a function directly on the Tracker struct.  Not really sure it needs to be an extension?